aws/aws-quick-ak.md

```diff
@@ -0,0 +1,222 @@
+## Amazon Web Services (AWS)
```

Comment:
```suggestion
## Amazon Web Services (AWS)
```

@webdev-ashishk can you please 
- update the name of the quiz
- and file name instead of `aws-quick-ak` to related 

thanks

Reply:
Sorry, AWS is not involved in this issue. It was my mistake

Reply:
please update or close this PR, thanks